win32/deluge-bbfreeze.py

```diff
@@ -19,11 +18,11 @@
 python_path += os.path.sep
 
 print "Python Path: %s" % python_path
-gtk_root = os.path.join(gtk.__path__[0], "..\\runtime\\")
+gtk_root = "C:\\Python27\\Lib\\site-packages\\gnome"
```

Comment:
We need the Gtk root folder to get locale files and I'm not sure this is it. Please add a comment.